src/components/input/input.tsx

```diff
@@ -280,6 +283,22 @@ export class Input
    */
   @Prop() multiple = false;
 
+  /**
+   * Specifies the type of content to help devices display an appropriate virtual keyboard.
+   * Read the native attribute's documentation on MDN for more info.
+   *
+   * @mdn [step](https://developer.mozilla.org/en-US/docs/Web/HTML/Global_attributes/inputmode)
```

Comment:
Should we be passing this @mdn tag through to the documentation site UI at all to indicate these are native html attributes?